app/controllers/CampaignApi.scala

```diff
@@ -36,6 +36,11 @@ class CampaignApi(override val wsClient: WSClient) extends Controller with Panda
     }
   }
 
+  def deleteCampaign(id: String) = APIAuthAction { req =>
+    CampaignRepository.deleteCampaign(id)
```

Comment:
Can we also delete the corresponding entries in the campaign content and notes tables, currently this will leave orphaned data in those tables. The orphaned data won't do any harm but removing it is cleaner.